src/test/scala/redis/RedisClusterTest.scala

```diff
@@ -0,0 +1,114 @@
+package redis
+
+
+import akka.util.ByteString
+import redis.api.clusters.ClusterSlots
+import redis.protocol.{DecodeResult, MultiBulk, RedisProtocolReply, RedisReply}
+
+import scala.concurrent.Await
+
+/**
+  * Created by npeters on 20/05/16.
+  */
+class RedisClusterTest extends RedisClusterClients {
+
+
+  var redisCluster:RedisCluster = null
+  override def setup(): Unit = {
+    super.setup()
+    redisCluster = RedisCluster(nodePorts.map(p=>RedisServer("127.0.0.1",p)))
+  }
+
+  sequential
+
+  "RedisComputeSlot" should {
+    "simple" in {
+      RedisComputeSlot.hashSlot("foo") mustEqual 12182
+      RedisComputeSlot.hashSlot("somekey") mustEqual 11058
+      RedisComputeSlot.hashSlot("somekey3452345325453532452345") mustEqual 15278
+      RedisComputeSlot.hashSlot("rzarzaZERAZERfqsfsdQSFD") mustEqual 14258
+      RedisComputeSlot.hashSlot("{foo}46546546546") mustEqual 12182
+      RedisComputeSlot.hashSlot("foo_312312") mustEqual 5839
+      RedisComputeSlot.hashSlot("aazaza{aa") mustEqual 11473
+    }
+  }
+
+  "clusterSlots" should {
+    "encoding" in {
+      val clusterSlotsAsByteString = ByteString(new sun.misc.BASE64Decoder().decodeBuffer("KjMNCio0DQo6MA0KOjU0NjANCiozDQokOQ0KMTI3LjAuMC4xDQo6NzAwMA0KJDQwDQplNDM1OTlkZmY2ZTNhN2I5ZWQ1M2IxY2EwZGI0YmQwMDlhODUwYmE1DQoqMw0KJDkNCjEyNy4wLjAuMQ0KOjcwMDMNCiQ0MA0KYzBmNmYzOWI2NDg4MTVhMTllNDlkYzQ1MzZkMmExM2IxNDdhOWY1MA0KKjQNCjoxMDkyMw0KOjE2MzgzDQoqMw0KJDkNCjEyNy4wLjAuMQ0KOjcwMDINCiQ0MA0KNDhkMzcxMjBmMjEzNTc4Y2IxZWFjMzhlNWYyYmY1ODlkY2RhNGEwYg0KKjMNCiQ5DQoxMjcuMC4wLjENCjo3MDA1DQokNDANCjE0Zjc2OWVlNmU1YWY2MmZiMTc5NjZlZDRlZWRmMTIxOWNjYjE1OTINCio0DQo6NTQ2MQ0KOjEwOTIyDQoqMw0KJDkNCjEyNy4wLjAuMQ0KOjcwMDENCiQ0MA0KYzhlYzM5MmMyMjY5NGQ1ODlhNjRhMjA5OTliNGRkNWNiNDBlNDIwMQ0KKjMNCiQ5DQoxMjcuMC4wLjENCjo3MDA0DQokNDANCmVmYThmZDc0MDQxYTNhOGQ3YWYyNWY3MDkwM2I5ZTFmNGMwNjRhMjENCg=="))
```

Comment:
you are right but we need to upgrade the travis.yml from jdk1.7 to jdk1.8 first.